### PR TITLE
Add an example of a standalone extension in `package:devtools_extensions`

### PR DESCRIPTION
**Pull request by kenzieschmoll**
This is a new Flutter web app that also provides the built assets under the `extension/devtools/build` directory.

### CODE REVIEW

File: packages/devtools_extensions/example/packages_with_extensions/standalone_extension/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_extensions/example/packages_with_extensions/standalone_extension/lib/main.dart:1]: needs review

File: packages/devtools_extensions/example/packages_with_extensions/standalone_extension/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# standalone_extension
```
Comment (kenzieschmoll) [inline packages/devtools_extensions/example/packages_with_extensions/standalone_extension/README.md:1]: needs review

File: packages/devtools_extensions/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 * Bump `devtools_shared` dependency to `^10.0.0`.
```
Comment (kenzieschmoll) [inline packages/devtools_extensions/CHANGELOG.md:4]: needs review

File: packages/devtools_extensions/example/packages_with_extensions/standalone_extension/extension/devtools/config.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+name: standalone_extension
```
Comment (kenzieschmoll) [inline packages/devtools_extensions/example/packages_with_extensions/standalone_extension/extension/devtools/config.yaml:1]: review

File: packages/devtools_extensions/example/packages_with_extensions/standalone_extension/pubspec.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+name: standalone_extension
```
Comment (kenzieschmoll) [inline packages/devtools_extensions/example/packages_with_extensions/standalone_extension/pubspec.yaml:1]: review

**Review by CoderDake — APPROVED** [review]

**Pull request merged**